samples/tutorial/Tutorial1.md

```diff
@@ -22,6 +22,12 @@ Let's start by making a workflow and screen to back the welcome view.
 
 Start by creating a new workflow and screen by creating a new file with the [file templates](../../install-templates.sh), adding it to the `tutorial-base` module:
 
+___NOTE___
+
+_The install script would not work for windows platform. For windows user, please add the scripts from [file templates folder](../../fileTemplates) manually to Android Studio or Intellij Idea._
```

Comment:
```suggestion
_Windows OS users: Please add the scripts from [file templates folder](../../fileTemplates) directly to Android Studio or Intellij Idea._
```

Reply:
hi @steve-the-edwards fixed this note wording as suggested